edward/inferences.py

```diff
@@ -238,14 +246,14 @@ def print_progress(self, t, loss):
                 print(self.variational)
 
     def finalize(self):
```

Comment:
again, i'd make this optional based on whether feed_dicts are being used. starting and stopping threads all the time is not ideal for fast debugging